cmd/gardener-resource-manager/app/app.go

```diff
@@ -206,9 +207,10 @@ func run(ctx context.Context, log logr.Logger, cfg *config.ResourceManagerConfig
 			// use dynamic rest mapper for target cluster, which will automatically rediscover resources on NoMatchErrors
 			// but is rate-limited to not issue to many discovery calls (rate-limit shared across all reconciliations)
 			opts.MapperProvider = func(config *rest.Config, httpClient *http.Client) (meta.RESTMapper, error) {
-				return apiutil.NewDynamicRESTMapper(
+				return thirdpartyapiutil.NewDynamicRESTMapper(
```

Comment:
Can you link the related PR here (https://github.com/kubernetes-sigs/controller-runtime/pull/2425) with a TODO statement that this should be reverted to `apiutil` once this PR is available? Same for other occurrences